_posts/2021-05-26-registry-security.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "Securing Apicurio Registry 2.0 using Keycloak"
+date:   2021-05-28 12:00:00
+author: carles
+categories: blog registry security
+---
+
+
+Hey everyone, in Apicurio Registry 2.0.0.Final we've introduced support for some security concepts. In this blog post, I will try to explain them and show you how to configure them.
+
+Core Features
+===
+Essentially in this blog post, we will cover the following:
+
+* **Authentication based on Keycloak** - optionally protect the registry such that the API requires users to authenticate (BASIC and OAuth supported)
+* **Role based authorization** - when authentication is enabled, you can also enable this feature, and users will need to have at least one role of (sr-admin, sr-developer, sr-readonly)
+* **Creator-only authorization option** - option to prevent changes to artifacts unless the authenticated user is the user who originally created the artifact
+
+
+Keycloak Configuration
+===
+
+### Authentication based on Keycloak
+In order to secure Apicurio Registry you will need a Keycloak server up and running along with some specific configuration.
+Essentially, you will need to create a Realm, two clients, and some roles (if you want to use roles).  By default Apicurio Registry expects the name of the Realm to be `registry`.
```

Comment:
Suggest "Essentially, you must create a Keycloak realm, ..."  ("realm" should be lowercase throughout this doc)

In general, throughout this doc, suggest "you must" instead of "you need" or "you will need to", and "you can" if step is optional.